CHANGELOG-3.1.md

```diff
@@ -13,7 +13,8 @@
 
 - Features:
   - [CoreBundle] Add `Zikula\Bundle\CoreBundle\Configurator` for writing config files to the filesystem (#4433).
-  - [extensions] Add StaticContent module to manage all static content (#4369).
+  - [FormExtensionsBundle] Add bsCustomFileInput for direct file selection feedback (#4491).
+  - [Extensions] Add StaticContent module to manage all static content (#4369).
```

Comment:
was not capitalized because the new module is going into the `/extensions` directory, but is not _named_ "Extensions". like "config" below.